docs/CONTRIBUTING.md

```diff
@@ -104,3 +105,28 @@ Development standards will be rigorously enforced on this project. Drupal mainta
 - [Twig coding standards](https://www.drupal.org/docs/develop/coding-standards/twig-coding-standards)
 - [JavaScript coding stand](https://www.drupal.org/docs/develop/standards/javascript)
 - [CSS Architecture](https://www.drupal.org/docs/develop/standards/css/css-architecture-for-drupal-8)
+
+### Linting
+
+To run the linting, you'll need to create a Github Personal Access token to be able to pull down the Yale package.
+
+- Visit the [Github instructions for creating a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+- Give it "Write packages" permissions. This will also check the repo permissions.
+- Export the token from your ~/.bash_profile or ~/.bashrc file by adding:
+  `export YALESITES_BUILD_TOKEN=[token]`
+- Source the file with `source ~/.bash_profile` or `source ~/.bashrc`.
```

Comment:
I wonder if we want to put this all here, or link to the Prerequisites section of the [Linting and Formatting repo](https://github.com/yalesites-org/eslint-config-and-other-formatting#installation)? That one has a link to help people who use other shells, and I consider it to be "canonical." If we ever changed the requirements, I'd like to only have to do it in one place...

Reply:
I actually think this should probably be in documentation somewhere more general under a heading like "Setting up your machine to install YaleSites packages" since it also applies to the theme, and anything else that we eventually package as a GitHub package.

Reply:
I don't believe we need any of this. Since we are pulling public packages, these steps should be unnecessary.

Reply:
They would be unnecessary if we were hosting on npm. To install GitHub packages, however - even public ones, requires a personal access token